includes/MirahezeMagicHooks.php

```diff
@@ -277,4 +277,31 @@ public static function onUserGetRightsRemove( User $user, array &$aRights ) {
 			}
 		}
 	}
+
+	public static function onSiteNoticeAfter( &$siteNotice, $skin ) {
+		$config = MediaWikiServices::getInstance()->getConfigFactory()->makeConfig( 'mirahezemagic' );
+		$wiki = new RemoteWiki( $config->get( 'DBname' ) );
```

Comment:
I think using this would be a performance issue? It would run on every request.

Reply:
I don't think it'd be much of a performance issue tbh, it's done like this in many other places throughout, although maybe it's not run on every page load there, so should it be converted back to `$cw*` variables then?

Reply:
yes, I think we should use the variable.

Reply:
John told me he suggested something so you should follow what he says.

Reply:
Yes, the way it is now is what I thought John was suggesting, unless I did it wrong/there is another way?

Reply:
Nope, this is fine. John says there wont be any performance issues.

Reply:
Thanks!